content/en/docs/setup/independent/high-availability.md

```diff
@@ -1,536 +1,502 @@
-                        "server auth",
-                        "client auth"
-                    ]
-                }
-            }
-        }
-     }
-     EOF
-     ```
-     ```bash
-     cat >ca-csr.json <<EOF
-     {
-        "CN": "etcd",
-        "key": {
-            "algo": "rsa",
-            "size": 2048
-        }
-     }
-     EOF
-     ```
+{{% capture steps %}}
 
-    {{< note >}}
-    **Optional:** You can modify `ca-csr.json` to add a section for `names`.
-    See [the CFSSL wiki](https://github.com/cloudflare/cfssl/wiki/Creating-a-new-CSR) for an example.
-    {{< /note >}}
+## First steps for both methods
 
-1. Next, generate the CA certs:
-     ```bash
-     cfssl gencert -initca ca-csr.json | cfssljson -bare ca -
-     ```
+{{< note >}}
+**Note**: All commands in this guide on any control plane or etcd node should be
+run as root.
+{{< /note >}}
 
-### Generate etcd client certs
+1. Find your pod CIDR. For details, see [the CNI network
+   documentation](/docs/setup/independent/create-cluster-kubeadm/#pod-network).
+   The example uses Calico, so the pod CIDR is `192.168.0.0/16`.
+1. Enable ssh-agent on your main device that has access to all other nodes in
+   the system by running `eval $(ssh-agent)`.
+1. Add your ssh identity to the session `ssh-add ~/.ssh/path_to_private_key`.
+1. When you ssh to any node make sure you add the `-A` flag like this `ssh -A 10.0.0.7`
+1. When using sudo on any node make sure to preserve the environment so the ssh
+   forwarding works, `sudo -E -s`.
+1. SSH between nodes to check that the connection is working properly.
+1. Create an apiserver load balancer with a DNS resolvable name.
 
-Generate the client certificates. While on `etcd0`, run the following:
+{{< note >}}
+**Note**: There are many configurations for load balancers. We have picked one
+for this guide for the sake of an example, but be aware this is highly
+customizable.
+{{< /note >}}
 
-```bash
-cat >client.json <<EOF
-{
-  "CN": "client",
-  "key": {
-      "algo": "ecdsa",
-      "size": 256
-  }
-}
-EOF
+In a cloud environment you want to put your control planes behind a tcp
+forwarding load balancer. This load balancer will distribute traffic to all
+healthy control plane nodes in its target list. The health check to determine if
+an api server is healthy or not is a TCP check on the port the kube-apiserver is
+listening on (default is `:6443`).
+
+The load balancer must have a resolvable DNS name (we do not suggest using an IP
+address directly in a cloud environment). The load balancer must have the ability to
+communicate with all control plane nodes on the apiserver port and it must allow
+incoming traffic on its listening port.
+
+After this step, you have a resolvable DNS name for your load balancer.
+
+You should add the control plane nodes to the load balancer but be aware they
+will fail the health check until the kube-apiserver is running.
+
+## Stacked control plane nodes
```

Comment:
@chuckha @Bradamant3 would it make sense (and is it even possible) to hide the two approaches here behind tabs, I think it would make the flow a bit better since there are common before/after steps for both.